### PR TITLE
N°5523 - Setup wizard : use the ITOP_APPLICATION constant instead of hardcoded "iTop" string

### PR DESCRIPTION
**Pull request by xtophe38**
These changes replace, where it makes sense, the usage of iTop as a product name to the variable ITOP_APPLICATION as it is already done in most of the setup process.

### CODE REVIEW

File: setup/wizardsteps.class.inc.php [hunk 1]
```diff
@@ -658,8 +658,9 @@ public function Display(\WebPage $oPage)
 			);
 			if ($oMutex->IsLocked())
 			{
+                $sCronMessage = '<div class="message">An '.ITOP_APPLICATION.' cron process is being executed on the target database. '.ITOP_APPLICATION.' cron process will be stopped during the setup execution.</div>';
 				$oPage->add(<<<HTML
-<div class="message">An iTop cron process is being executed on the target database. iTop cron process will be stopped during the setup execution.</div>
+$sCronMessage
 HTML
 				);
```
Comment (Hipska) [inline setup/wizardsteps.class.inc.php]: There is no added benefit for adding the text in a var before adding in another var.

I would just do this: `$oPage->add($sCronMessage)` or leave it as a single line without the intermediate var.

**Review by piRGoif — CHANGES_REQUESTED** [review]
Same comment as Hipska :)

File: setup/wizardsteps.class.inc.php [hunk 2]
```diff
@@ -658,8 +658,9 @@ public function Display(\WebPage $oPage)
 			);
 			if ($oMutex->IsLocked())
 			{
+                $sCronMessage = '<div class="message">An '.ITOP_APPLICATION.' cron process is being executed on the target database. '.ITOP_APPLICATION.' cron process will be stopped during the setup execution.</div>';
```
Comment (Molkobain) [inline setup/wizardsteps.class.inc.php]: It's a bit sad to rollback from a clear HEREDOC to a PHP concatenated string, I would rather keep the HEREDOC.

Comment (Hipska) [reply setup/wizardsteps.class.inc.php]: @Molkobain How to add a STATIC inside a HEREDOC?

**Comment by piRGoif** [comment]
@xtophe38 can you do the asked changes ?

**Review by Hipska — APPROVED** [review]

**Review by piRGoif — APPROVED** [review]

File: setup/wizardsteps.class.inc.php (hunk 2 above)
Comment (piRGoif) [reply setup/wizardsteps.class.inc.php]: Discussed with @xtophe38 directly : the constant cannot remain in heredoc, so we choose to adopt Hipska suggestion.
Closing the conversation

File: setup/wizardsteps.class.inc.php (hunk 1 above)
Comment (piRGoif) [reply setup/wizardsteps.class.inc.php]: That what was done in the last commit :) 
Closing conversation.

**Review by Hipska — APPROVED** [review]
@piRGoif I already reviewed and approved 😉

**Comment by piRGoif** [comment]
> @piRGoif I already reviewed and approved 😉

Wooops sorry !
Thanks for the review !

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Accepted in functional review, will be part of the upcoming 2.7.8 / 3.1.0

File: setup/wizardsteps.class.inc.php [hunk 3]
```diff
@@ -658,10 +658,7 @@ public function Display(\WebPage $oPage)
 			);
 			if ($oMutex->IsLocked())
 			{
-				$oPage->add(<<<HTML
-<div class="message">An iTop cron process is being executed on the target database. iTop cron process will be stopped during the setup execution.</div>
-HTML
-				);
+				$oPage->add('<div class="message">An '.ITOP_APPLICATION.' cron process is being executed on the target database. '.ITOP_APPLICATION.' cron process will be stopped during the setup execution.</div>');
```
Comment (Molkobain) [inline setup/wizardsteps.class.inc.php]:
```suggestion
				$oPage->add('<div class="message">'.ITOP_APPLICATION.' cron process is being executed on the target database. '.ITOP_APPLICATION.' cron process will be stopped during the setup execution.</div>');
```

**Comment by Molkobain** [comment]
@xtophe38 next steps are:
  * Create a ticket in our bug tracker
  * Add ticket number in the PR title (`N°XXXX - Setup wizard : use the ITOP_APPLICATION constant instead of hardcoded "iTop" string`)
  * Merge PR using "Squash & merge" as there are several commits and it's not relevant to keep them separated
  * Merge support/2.7 into support/3.0 into develop
  * Resolved ticket as usual

Feel free to come to us for a copilot 👨‍✈️

**Pull request merged**